plugins/QnA/class.qna.plugin.php

```diff
@@ -1317,7 +1317,7 @@ public function discussionsApiController_normalizeOutput(
         }
 
         $discussion['attributes']['question'] = [
-            'status' => strtolower($discussion['qnA']),
+            'status' => isset($discussion['qnA']) ? strtolower($discussion['qnA']) : 'unanswered',
```

Comment:
```suggestion
            'status' => empty($discussion['qnA']) ? strtolower($discussion['qnA']) : 'unanswered',
```

I suggest `empty` over `isset` because it handles one more potential data error. I just fixed a different bug with this exact case so I may be a bit more paranoid.